Assets/Scripts/ShadowPuzzle/PilarCheck.cs

```diff
@@ -0,0 +1,32 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor.Experimental.GraphView;
+using UnityEngine;
+
+public class PilarCheck : MonoBehaviour
+{
+    public bool pilarInPlace;
+    public GameObject pilarPlace;
+    public GameObject pilar;
+
+    void OnTriggerEnter(Collider other)
```

Comment:
```suggestion
    private void OnTriggerEnter(Collider other)
```
altijd access modifier hebben